core_lib/src/interface/editor.cpp

```diff
@@ -516,15 +516,22 @@ void Editor::copy()
 
     if (currentLayer->hasAnySelectedFrames() && !select()->somethingSelected()) {
         clipboards()->copySelectedFrames(currentLayer);
+        return;
     }
-    else if (currentLayer->type() == Layer::BITMAP) {
+
+    if (currentLayer->type() == Layer::BITMAP) {
         BitmapImage* bitmapImage = static_cast<BitmapImage*>(currentLayer->getLastKeyFrameAtPosition(currentFrame()));
         clipboards()->copyBitmapImage(bitmapImage, select()->mySelectionRect());
+        return;
```

Comment:
I personally think 'else if' is more suitable here than abrupt return statements that are easily forgotten or removed in code alterations.

Reply:
Hmm, I generally prefer an early return because that makes it very clear that nothing else is gonna happen for that condition anymore so I no longer have to keep that branch in my mind beyond that point. But now that I think about it, in this particular case a switch statement would probably be even more fitting.